.github/workflows/main.yml

```
@@ -38,21 +38,23 @@ jobs:
-          AWS_EC2_METADATA_DISABLED: true
         run: |
           aws s3 sync \
             --region us-east-1 \
             dist s3://klkl-frontend \
-            --delete
+            --delete \
+
+      - name: Create Invalidation
+        env:
+          CLOUD_FRONT_ID: ${{ secrets.AWS_CLOUDFRONT_ID}}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:21: Double quote to prevent globbing and word splitting

(shellcheck)